docs/application/native/guides/ui/dali/capture.md

```diff
@@ -0,0 +1,67 @@
+# Capture
+
+
+Capture snapshot the current DALi scene. The captured result can be saved as a file and also can be used to be rendered by DALi `ImageView` or `ImageVisual`.
+
+In this tutorial, we show how to capture scene and to use it.
+
+
+## Tutorial
+
+Applications should follow the example below to use `Dali::Capture`.
+
+``` C++
+Capture capture = Capture::New();
+
+capture.Start( source, size, path, clearColor, quality);
+```
+
+Each parameter of `Capture::Start` is denoted as follows:
+
+ - **Actor source**: source actor to be rendered scene. Only this source actor and its children are captured.
```

Comment:
Not clear, please rephrase, is it?
- **Actor source**: source actor to be rendered on the screen. Only this source actor and its children are captured.

Reply:
Thank you for your review.
I fixed it, but the actor is to capture, so I modified it as follows

source actor to be captured. Only this source actor and its children are captured.